### PR TITLE
Added Setup instructions for Ubuntu/Debian

### LINKED ISSUE #7: Please keep instructions on Readme.md
For now Readme.md is not super long and we don't need separate documents. Please merge the setup info to the Readme.

Once other documents are required, please follow best practices. Keep docs in:
* /docs

Keep images in:
* /docs/images

### PR DESCRIPTION
**Pull request by betterclever**
Fixes #7 , #3 
Changes:
* Setup instruction included in README.md itself.
* Add setup instructions for Ubuntu/Debian.

### CODE REVIEW

**Comment by betterclever** [comment]
@chiragw15 @mayank408 @DravitLochan Can you test and confirm if it works fine on your system? I could not test on Ubuntu, since I am using Arch Linux. 
I have tested installation on a Cloud Server though and it should be fine on a normal system.

File: README.md [hunk 1]
```diff
@@ -19,5 +19,76 @@ It will enable you to bring Susi AI intelligence to all devices you may think li
...
+* Install Python Version: 3.5+
+    * ```sudo apt install python3```
+    * ```sudo apt install python3-pip```
+
+* Install Festival TTS
+    * ```sudo apt install festival```
+    
+* Install CMU_US Voices
+     ```
+     # Using Package from Arch Linux Repository to extract Voice Model. It works well on Ubuntu.
```
Comment (chiragw15) [inline README.md]: Is this line intended to be inside code block?

**Comment by mariobehling** [comment]
@betterclever Please take this serious and ensure this works on Debian and Ubuntu. I am sure, how can test this running a VM etc. Please take responsibility and don't drop this on others without testing yourself. Thank you.

File: README.md (hunk 1 above)
Comment (betterclever) [inline README.md]: It is a comment. Ignore it.

**Comment by betterclever** [comment]
@mariobehling Actually, this needs Audio driver access, so it wasn't easily possible on VM. I'll install Ubuntu on a different hard drive and test.

**Pull request merged**